tools/tools.go

```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build tools
-// +build tools
+//go:build go1.18
+// +build go1.18
```

Comment:
We should also add check.go and old.go then

Reply:
Done